WMFPageHistorySection.h

```diff
@@ -0,0 +1,17 @@
+//
+//  WMFPageHistorySection.h
+//  Wikipedia
+//
+//  Created by Nick DiStefano on 4/3/16.
+//  Copyright © 2016 Wikimedia Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@class WMFPageHistoryRevision;
+
+@interface WMFPageHistorySection : NSObject
+
+@property (nonatomic, copy, readwrite) NSString* _Nullable sectionTitle;
```

Comment:
Can we make these properties readonly and pass them on init so the class is immutable?